### PR TITLE
BSPIMX8M-3300 Wip/bhahn/finalize bsp doc

### PR DESCRIPTION
**Pull request by BHahn42618**
Add BSP Manual for NXP PD24.1.0 release
Add NXP PD24.1.0 release to supported releases in yocto scarthgap manual
correct spelling mistake

### CODE REVIEW

File: source/bsp/dt-overlays.rsti [hunk 1]
```diff
@@ -1,3 +1,17 @@
+Otherwise you can show the content of the fitImage including all overlay
+configs in the fitImage with this command in Linux:
```
Comment (ymoog) [inline source/bsp/dt-overlays.rsti:2]: This file is included in older releases. Does it make sense to refer to a FIT image?

File: source/conf.py [hunk 2]
```diff
@@ -159,6 +159,14 @@
         'manual',
         False,
     ),
+    (
+        'bsp/imx8/imx8mp/pd24.1.0_nxp',
+        'imx8mp-pd24.1.0_nxp.tex',
+        'i.MX 8M Plus BSP Manual PD24.1.0 NXP',
```
Comment (ymoog) [inline source/conf.py:165]: Funny that of all things, the peculiar release naming scheme falls apart in the documentation.

Comment (BHahn42618) [reply source/conf.py:165]: Is this something, where you think I should change somthing, or is this just an observation? If you think I should change anything. What would be your suggestion?

Comment (ymoog) [reply source/conf.py:165]: Just an observation. I like your solution.

**Review by tremmet — DISMISSED** [review]
Thanks!

**Review by tremmet — DISMISSED** [review]

File: source/bsp/imx8/imx8mp/imx8mp.rst [hunk 3]
```diff
@@ -22,6 +22,16 @@ Mainline HEAD
...
 
+BSP-Yocto-NXP-i.MX8MP-PD24.1.0
+==============================
+
+.. toctree::
+   :caption: Table of Contents
+   :numbered:
+   :maxdepth: 2
+
+   pd24.1.0_nxp
```
[the remark this answers is not in the document]
Comment (tremmet) [reply source/bsp/imx8/imx8mp/imx8mp.rst:33]: Jonas this is a management decision. We have a BSP-Yocto-Ampliphy-i.MX8MP-PD24.1.0 and a BSP-Yocto-NXP-i.MX8MP-PD24.1.0.

File: source/bsp/imx8/imx8mp/pd24.1.0_nxp.rst [hunk 4]
```diff
@@ -0,0 +1,591 @@
...
+.. _dl-server: https://download.phytec.de/Software/Linux/BSP-Yocto-i.MX8MP/
+.. _dl-sdk: https://download.phytec.de/Software/Linux/BSP-Yocto-i.MX8MP/BSP-Yocto-NXP-i.MX8MP-PD24.1.0/sdk/ampliphy-vendor-xwayland/
+.. |link-image| replace:: https://download.phytec.de/Software/Linux/BSP-Yocto-i.MX8MP/BSP-Yocto-NXP-i.MX8MP-PD24.1.0/images/ampliphy-vendor-xwayland/phyboard-pollux-imx8mp-3/phytec-qt6demo-image-phyboard-pollux-imx8mp-3.rootfs.wic.xz
+.. |link-partup-package| replace:: https://download.phytec.de/Software/Linux/BSP-Yocto-i.MX8MP/BSP-Yocto-NXP-i.MX8MP-PD24.1.0/images/ampliphy-vendor-xwayland/phyboard-pollux-imx8mp-3/phytec-qt6demo-image-phyboard-pollux-imx8mp-3.rootfs.partup
+.. |link-boot-tools| replace:: https://download.phytec.de/Software/Linux/BSP-Yocto-i.MX8MP/BSP-Yocto-NXP-i.MX8MP-PD24.1.0/images/ampliphy-vendor-xwayland/phyboard-pollux-imx8mp-3/imx-boot-tools/
+.. |link-bsp-images| replace:: https://download.phytec.de/Software/Linux/BSP-Yocto-i.MX8MP/BSP-Yocto-NXP-i.MX8MP-PD24.1.0/images/ampliphy-vendor-xwayland/phyboard-pollux-imx8mp-3/
+.. _releasenotes: https://git.phytec.de/phy2octo/tree/releasenotes?h=imx8mp
+.. _`static-pdf-dl`: ../../../_static/imx8mp-head.pdf
```
[the remark this answers is not in the document]
Comment (BHahn42618) [reply source/bsp/imx8/imx8mp/pd24.1.0_nxp.rst]: Changed it

**Review by jonas-rem — APPROVED** [review]

**Pull request merged**